doc/examples.xml

```diff
@@ -309,3 +309,45 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="BananaTreeGraph">
+<ManSection>
+  <Oper Name="BananaTreeGraph" Arg="[filt, ]m, n"/>
+  <Returns>A digraph.</Returns>
```

Comment:
Unless this is a standard for Digraphs, no dot is needed for `<Returns>` elements.